libs/ledger-live-common/src/api/Ethereum.test.ts

```diff
@@ -78,6 +78,10 @@ describe("apiForCurrency", () => {
         expect(
           (formatUrlInput.query as ParsedUrlQueryInput).from_height
         ).toEqual(blockHeightMock);
+        // ↓ this is necessary in order to get transactions after the block height and not before. Order is still descending in the end
+        expect((formatUrlInput.query as ParsedUrlQueryInput).order).toEqual(
```

Comment:
Shouldn't this be modified in "should use descending order" test instead ?

Reply:
No both modifications are necessary for the same behaviour. 
-> If you're using the`from_height` param, you need to have the `order=ascending` param as well.
No reason to have it as a seperated test.

Reply:
I am talking about the one before it